packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -107,14 +109,14 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
             "delete" -> onDelete(result, data)
             "save" -> onSave(result, data)
             "clear" -> onClear(result)
-            "setupObserve" -> onSetupObserve(result)
-            "configureModelProvider" -> onConfigureModelProvider(result, data)
+            "setUpObserve" -> onSetUpObserve(result)
```

Comment:
nit: I think this was okay as 'setup' because it is generally considered a single word.

Reply:
Sorry this change was kind of driven by my nitpicky desire for semantic "accuracy" -
_setup_ as a single word is generally a noun whereas _set up_ is the verb. In this case, I felt the verb was technically more accurate

Reply:
haha - fair enough.